{{cookiecutter.project_name}}/docker-compose.override.yml

```diff
@@ -18,3 +18,7 @@ services:
     volumes:
       # We only mount source code in development:
       - .:/code
+    # set env variables. See: https://github.com/wemake-services/wemake-django-template/issues/1518
```

Comment:
What about production env?

Reply:
It's not required for production. Because we do not mount current directory to docker container.

Reply:
Please, add this line to the comment docs 🙂